driver/src/main/java/org/neo4j/driver/SessionConfig.java

```diff
@@ -246,16 +247,38 @@ public Builder withDefaultAccessMode(AccessMode mode) {
         }
 
         /**
-         * Set the database that the newly created session is going to connect to.
+         * Set database name for queries to be executed within session. This also applies to downstream query
+         * executions, like transactions started by session.
```

Comment:
```suggestion
         * executions, like transactions started by the session.
```

Reply:
It is actually a reference to session in a general sense, which is why `the` is not used. I will see if this can be expressed in a better way.